package.json

```diff
@@ -9,12 +9,22 @@
   },
   "dependencies": {
     "@expo/vector-icons": "^12.0.3",
+    "@react-native-async-storage/async-storage": "^1.13.4",
+    "@react-native-community/masked-view": "^0.1.10",
+    "@react-navigation/drawer": "^5.12.3",
+    "@react-navigation/native": "^5.9.2",
+    "@react-navigation/stack": "^5.14.2",
     "expo": "~40.0.0",
     "expo-status-bar": "~1.0.3",
     "native-base": "^3.0.0-next.19",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
+    "react-native-gesture-handler": "^1.8.0",
+    "react-native-onboarding-swiper": "^1.1.4",
```

Comment:
Why did we need this? Can we not implement this manually?

Reply:
I am using only ScrollView for swiper Screen